operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainStatus.java

```diff
@@ -116,28 +117,24 @@ public DomainStatus(DomainStatus that) {
    */
   public DomainStatus addCondition(DomainCondition newCondition) {
     if (conditions.contains(newCondition)) {
-      conditions = conditions.stream()
```

Comment:
If we have a sequence of Failed/progressing/failed, the progressing condition remains because of this change. Is this behavior change intentional?

Reply:
Yes - since we can have multiple failures, it is not a given that we want to end the progressing state at that point. It will be removed when we hit the end progressing step.

Reply:
ok